templates/terraform/examples/base_configs/tutorial.md.erb

```diff
@@ -0,0 +1,80 @@
+<% autogen_exception -%>
+<%= "# #{example.name.camelize(:upper).uncombine} - Terraform" %>
+
+## Setup
+
+<walkthrough-author name="rileykarson@google.com" analyticsId="UA-125550242-1" tutorialName="<%= "#{example.name.underscore}" %>" repositoryUrl="https://github.com/terraform-google-modules/docs-examples"></walkthrough-author>
+
+Welcome to Terraform in Google Cloud Shell! We need you to let us know what project you'd like to use with Terraform.
+
+<walkthrough-project-billing-setup></walkthrough-project-billing-setup>
+
+Terraform provisions real GCP resources, so anything you create in this session will be billed against this project.
+
+## Terraforming!
+
+Let's use {{project-id}} with Terraform! Click the Cloud Shell icon below to copy the command
+to your shell, and then run it from the shell by pressing Enter/Return. Terraform will pick up
+the project name from the environment variable.
+
+```bash
+export GOOGLE_CLOUD_PROJECT={{project-id}}
```

Comment:
I thought this was the variable that cloud shell fills in automatically. Does it not, or does OiCS not open into a particular project?

Reply:
Cloud Shell normally *does*, but Open in Cloud Shell doesn't - it isn't clear which project would be in use, unlike in Cloud Console where you have almost always clearly selected a project. I'm working on getting a better experience here, but it isn't there quite yet.